include/xtypes/Module.hpp

```diff
@@ -357,18 +357,18 @@ class Module
         return result.second;
     }
 
-    AliasType& alias(
+    const AliasType& alias(
```

Comment:
It can be mutable

Reply:
This cannot be done effectively without removing the `const` attribute from the `PairModuleSymbol<const Module*, ...>` declaration. @LuisGP is there any reason why `const` is being used there? This would also let to remove several `const_cast` operations, as suggested in https://github.com/eProsima/xtypes/pull/52#discussion_r361996940.

Reply:
`PairModuleSymbol` is intended to be used as an auxiliar pair to check and inspect modules, not to modify them. That why its module is `const`. 
It is true that it is an internal alias and we could return a non-const pointer, but then, the "trick" should be done with a non-const `resolve_scope` method, which would be dirty, I think.

Reply:
Now there are two versions, `const` and `non-const` respectively, for the `alias` method.